ipn/ipnlocal/local.go

```diff
@@ -4669,7 +4673,7 @@ func (b *LocalBackend) setNetMapLocked(nm *netmap.NetworkMap) {
 
 	if b.tailFSSharingEnabledLocked() {
 		b.updateTailFSPeersLocked(nm)
-		b.tailFSNotifyCurrentSharesLocked()
+		b.tailFSNotifyCurrentSharesOnceLocked()
```

Comment:
The reason we can't do this [here](https://github.com/tailscale/tailscale/blob/main/ipn/ipnlocal/local.go#L2285) is because on the request for initial information, we have not yet gotten the node attribute that informs us that TailFS sharing is enabled.